test/models/test_auto_models.py

```diff
@@ -374,17 +383,10 @@ async def test_ngc_unsupported_operations():
         await fs.find("some/path", maxdepth=1)
 
 
+# Test automodel is working correctly for lightweight model
 @pytest.mark.parametrize(
     "model_class",
-    [
```

Comment:
Did you mean to remove all these or was this just for debugging.

Reply:
I did remove them on purpose, just testing with one model is sufficient for this